src/bin/ion/commands/beta/generate/utils.rs

```diff
@@ -121,8 +115,8 @@ impl Language for RustLanguage {
         "rust".to_string()
     }
 
-    fn file_name_for_type(name: &str) -> String {
-        name.to_case(Case::Snake)
+    fn file_name_for_type(_name: &str) -> String {
```

Comment:
(🗺️ PR tour) Since Rust code generation uses a single file for all generated data models this method outputs a constant file name `ion_generated_code`